client/src/pages/commonComponents/Breadcrumbs/Breadcrumbs.tsx

```diff
@@ -0,0 +1,46 @@
+import './Breadcrumbs.less';
+
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { Normaltekst } from 'nav-frontend-typografi';
+
+import person from './person.svg';
+
+export type Breadcrumb = {
```

Comment:
```suggestion
export interface BreadcrumbProps {
```
Vi bruker props som interface alle andre steder